src/python_testing/basic_composition_support.py

```diff
@@ -26,12 +26,30 @@
 from pprint import pformat, pprint
 from typing import Any, Optional
 
+import chip.clusters as Clusters
 import chip.clusters.ClusterObjects
 import chip.tlv
 from chip.clusters.Attribute import ValueDecodeFailure
 from mobly import asserts
 
 
+def arls_populated(tlv_data: dict[int, Any]) -> tuple[bool, bool]:
```

Comment:
returning a dataclass with named members has_arl/has_carl (or arl/carl) would be easier to follow than two bools.

Reply:
I have a followup ready, but given that it's now 9:30 and we're well past the SVE deadline, and the CI takes multiple hours, I'm going to resolve this and address in a follow up PR. PR will be up in a few minutes.